season_pass/main.py

```diff
@@ -36,6 +36,8 @@ def handle_exceptions(e: Exception):
         status_code = HTTP_404_NOT_FOUND
     elif type(e) in (InvalidSeasonError, NotPremiumError):
         status_code = HTTP_400_BAD_REQUEST
+    elif type(e) in (ServerOverloadError):
```

Comment:
comma needed after err name unless you cannot make tuple with one element

Reply:
done